File: findYear.py
import datetime
name = input("Please enter your name: ")
age = int (input("Please enter your age: "))

today = datetime.date.today()
thisYear = today.year

anwser = thisYear - age + 100

print("Hello,", name, "you will be 100 years old in the year:", anwser)

File: findYear.php
<?php

$name = readline("Please enter your name: ");
$age = readline("Please enter your age: ");

$today = date("Y");

$answer = $today - $age + 100;

echo ("\nHello, " . $name . " you will be 100 years old in the year: " . $answer . "\n");
